library-management-system-frontend/src/app/components/home/home.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { IssueBook } from 'src/app/models/issue-book';
+import { IssueBookService } from 'src/app/services/issue-book.service';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.css']
+})
+export class HomeComponent implements OnInit {
+issuedDetails:IssueBook[]=[];
+i?:number
```

Comment:
Define variable by their scope. in this place i is function scope not class scope